packages/babel-generator/src/node/parentheses.js

```diff
@@ -219,6 +220,13 @@ export function ConditionalExpression(node: Object, parent: Object): boolean {
   return UnaryLike(node, parent);
 }
 
+export function OptionalMemberExpression(
+  node: Object,
+  parent: Object,
+): boolean {
+  return t.isMemberExpression(parent);
```

Comment:
Aren't parentheses also needed when the parent is a `CallExpression`? (`(a?.b)()`)

Reply:
Updated!